front-end/src/components/molecules/side-bar/side-bar.tsx

```diff
@@ -15,6 +19,21 @@ const dashboardElements = [
 ];
 
 export default function Sidebar() {
+  const router = useRouter();
+
+  const handleLogout = () => {
+    localStorage.removeItem("token")
```

Comment:
@solonkonora, I emphasised you use the `tokenService` object, to handle clearing the session token from localStorage.
I bet you this will not do what we want to.
- `token` is not even the key in the localStorage.
-  Just as we used `tokenService` in the `LoginOrSignupPage` to set, check the service, and see how to clear the token